fuel/app/config/theme.php

```diff
@@ -35,9 +35,7 @@
 	 * The theme search paths.  They are searched in the order given.  You can add paths
 	 * on the fly via Theme::add_path($path) or Theme::add_paths(array($path1, $path2));
 	 */
-	'paths' => array(
-		APPPATH.'themes',
-	),
+	'paths' => explode(',', APPPATH.'themes'.','.($_ENV['THEME_PACKAGE'] ? PKGPATH.$_ENV['THEME_PACKAGE'] : '')),
```

Comment:
We don't have `THEME_PACKAGE` currently defined in the env configs on this branch, is it required?